doc/api/process.md

```diff
@@ -472,8 +472,8 @@ added: 6.4.0
 
 * {string}
 
-The `process.argv0` property stores a read-only copy of the original value of
-`argv[0]` passed when Node.js starts.
+`process.argv0` is a configurable property that stores a read-only copy of the
+original value of `argv[0]` passed when Node.js starts.
```

Comment:
Seems like this is wordier than it needs to be. Both read-only _and_ configurable?